eslint-rules/data-testid/require-data-testid.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description:
+        'Enforce that required elements have a data-testid attribute for testing purposes.',
+      recommended: true,
+    },
+    fixable: null,
+    schema: [],
+  },
+  create(context) {
+    return {
+      JSXOpeningElement(node) {
+        if (node.name.name === 'div') {
```

Comment:
What do you say we change this to 'button' instead of 'div' ?

Reply:
Sounds good. I can add that change.